cmd/swarm-rafttool/main.go

```diff
@@ -5,6 +5,8 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/docker/swarmkit/cmd/swarmd/defaults"
+
```

Comment:
minor nit: We generally don't have blank lines between non-standard-library imports.